src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionIntroViewController.swift

```diff
@@ -26,25 +26,25 @@ class ExposureSubmissionIntroViewController: DynamicTableViewController, Exposur
 
 	// MARK: - View lifecycle methods.
 
+	override func viewDidLoad() {
+		super.viewDidLoad()
+
+		navigationFooterItem?.primaryButtonTitle = AppStrings.ExposureSubmission.continueText
+
+		setupView()
+		setupBackButton()
+	}
+
 	override func viewWillAppear(_ animated: Bool) {
 		super.viewWillAppear(animated)
 
-		// The button is shared among multiple controllers,
-		// make sure to reset it whenever the view appears.
-		setButtonTitle(to: AppStrings.ExposureSubmission.continueText)
-		button?.accessibilityIdentifier = "AppStrings.ExposureSubmission.continueText"
+		footerView?.primaryButton?.accessibilityIdentifier = "AppStrings.ExposureSubmission.continueText"
```

Comment:
General remark: Wouldn't it be easier to have all accessibilityIdentifiers bundled in a centralized file, instead of hardcoding it each and every time?

Reply:
Please raise this to @i336616.

Reply:
Talked to @i336616 and @haosap about this. #707 will tackle this issue. Consider this comment as resolved.